packages/govuk-frontend/src/govuk/components/footer/footer.yaml

```
@@ -49,11 +49,11 @@ params:
       - name: width
         type: string
         required: false
-        description: Width of each navigation section in the footer. Defaults to full width. You can pass any design system grid width here, for example, 'one-third'; 'two-thirds'; 'one-half'.
+        description: Width of each navigation section in the footer. Defaults to `"full"`. You can pass any design system grid width here, for example, `"one-third"`; `"two-thirds"`; `"one-half"`.
```

Is there a reason for using semi-colons here, when we use a standard comma separated list for other sets of options? Is it because it's non-exhaustive?

If not, can we change it?

```suggestion
        description: Width of each navigation section in the footer. Defaults to `"full"`. You can pass any design system grid width here, for example, `"one-third"`, `"two-thirds"` or `"one-half"`.
```

---

Mind if I put the default at the end like we do elsewhere?

Also adding the `–` before the values like your suggestion in https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1397385071

```suggestion
        description: Width of each navigation section in the footer. You can pass any Design System grid width here – for example, `"one-third"`, `"two-thirds"` or `"one-half"`. Defaults to `"full"`.
```